script/script.js

```diff
@@ -0,0 +1,17 @@
+ document.querySelector('#create_task').addEventListener('click', myClick);
+
+ function myClick(){
+    //console.log('work')
+    let text = document.querySelector('.i-1').value;
+    //console.log(a);
+    //document.querySelector('.out').innerHTML = ; тут іннер хтмл 
+    // він заміняє весь контент в пісі-какі-попі. мені треба 
+   
+
+   const piratik = document.createElement('li');
+   piratik.textContent = text;
+   const pandora = document.getElementById('ul');
```

Comment:
pandora does not make sense. What are you storing in this variable?
My guess is: taskListElement

Reply:
Thanks. Replaced to logigal names.